docs/guides/formulasC2/build-template.md

```diff
@@ -0,0 +1,260 @@
+---
+heading: Building Formulas
+seo: Build a Formula Template | Formulas | Cloud Elements API Docs
+title: Build a Formula Template
+description: How to build a formula template
+layout: sidebarleft
+apis: API Docs
+platform: formulas
+breadcrumbs: /docs/guides/home.html
+parent: Back to Guides
+order: 5
+---
+
+# Formula Templates
+
+Formula templates enable you to create a single template for a formula that you can reuse with different elements and values. After you create a formula template, you can then create a formula instance where you replace the formula variable with actual element instances and values. This approach helps you build efficient and reusable formulas.
+
+{% include callout.html content="<strong>On this page</strong></br><a href=#build-a-formula-template>Build a Formula Template</a></br><a href=#test-formula-templates>Test Formula Templates</a></br><a href=#create-variables>Create Variables</a></br><a href=#edit-and-delete-variables>Edit and Delete Variables</a></br><a href=#review-executions>Review Executions</a></br><a href=#copy-a-formula-template>Copy a Formula Template</a></br><a href=#parallel-executions>Parallel Executions</a></br><a href=#use-a-formula-as-a-resource>Use a Formula as a Resource</a></br><a href=#rename-a-formula-template>Rename a Formula Template</a></br><a href=#add-a-description-to-a-formula-template>Add a Description to a Formula Template</a></br><a href=#delete-a-formula-template>Delete a Formula Template</a></br><a href=#deactivate-a-formula-template>Deactivate a Formula Template
+</a>" type="info" %}
+
+## Build a Formula Template
```

Comment:
We may need a section for Importing/Exporting Formulas.  Paris is styling that feature now.  Should be available soon.